include/swift/SILOptimizer/Analysis/ActivityAnalysis.h

```diff
@@ -0,0 +1,235 @@
+//===--- ActivityAnalysis.h - Differentiable Activity Analysis --*- C++ -*-===//
```

Comment:
I think this file should be called `DifferentiableActivityAnalysis`. "Activity analysis" is not well-defined out of AD.

Reply:
Done.